### ISSUE TITLE
what's the relationship between socket.io and mqtt?

### ISSUE THREAD

**Issue by roy-mor** [Yaakov-Belch/mqtt-meetup#3]
I understand that socket.io is a library that uses WebSockets if it is available, and fallbacks to other technologies if not (long-polling, regular ajax etc).

If we used socket.io, what would we need mqtt for? I understand it's a pub-sub broker/technology, but couldn't we just use socket.io and some app code in order to get the functionality we saw at you demo at the meetup?

Thanks

**Comment by Yaakov-Belch** [comment]
Yes --- you can write reactive applications with socket.io and without mqtt.  You can also use raw websockets.
